src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +34,17 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+     *
+     * @param set of String
+     * @return the same set with all the strings in lowercase
+     */
+    public static Set<String> getStringInLowerCase(Set<String> set) {
```

Comment:
getStringsInLowerCase() would be more reflective than getStringInLowerCase() since you are passing a collection. Also helps immediately understand the difference from the built in toLowerCase(). Minor comment.